css-canvas-medical/styles.css

```diff
@@ -0,0 +1,223 @@
+.container {
+  width: 1000px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.column-50 {
+  width: 50%;
+  padding-left: 30px;
+}
+
+.row {
+  display: flex;
+}
+
+a {
+  color: rgb(195, 195, 195);
+  text-decoration: none;
+  font-family: 'Avenir', sans-serif, serif;
+}
+
+.hero-picture {
+  margin-top: 25px;
+}
+
+.nav {
+  width: 10%;
+  margin-left: 50px;
+  margin-top: 25px;
+}
+
+.column-15 {
+  width: 15%;
+  margin-top: 15px;
+}
+
+.column-40 {
+  width: 40%;
+  margin-top: 20px;
+}
+
+.contact-us {
+  background-color: rgb(41, 68, 99);
+  color: white;
+  padding: 10px 10px;
+  border-radius: 2px;
+  font-family: 'Avenir', sans-serif, serif;
+}
+
+.hero {
+  width: 95%;
+  position: relative;
+  margin-left: 20px;
+}
+
+.col-text {
+  position: absolute;
+  left: 250px;
+  width: 35%;
+  top: 150px;
+  font-family: 'Avenir', sans-serif, serif;
+  color: white;
+}
+
+.emr-text {
+  font-family: 'Tiempos Headline', sans-serif, serif;
+}
+
+.primary {
+  position: absolute;
+  top: 40px;
+  color: rgb(255, 255, 255);
+  font-family: 'Tiempos Headline', sans-serif, serif;
+}
```

Comment:
Why does this need to be absolute? `col-text` is already `absolute`. If you need more space between this element and its siblings, use `margin` or `padding`